README.md

```diff
@@ -539,8 +536,8 @@ your C apps with.
 ### Memory
 
 RAM starts at 0x4000\_0000 and ends at 0x4002\_0000 (128 KB). The app
-will be loaded by firmware at RAM start. The stack for the app is
-setup to start just below RAM end (see
+will be loaded by firmware at the top of RAM. The stack for the app is
+setup to start at the bottom of RAM (see
```

Comment:
I think using "top" and "bottom" here is confusing. Perhaps use:
- "The app will be loaded by firmware from 0x4000\_0000".
- "The stack for the app is setup to start just below the highest address".

Reply:
If the aim is to avoid using "top" and "bottom", I think it would also be best to avoid "below" and "highest".

Now that I think about it, maybe the original wording wasn't that bad.

Maybe:
```
RAM starts at 0x4000\_0000 and ends at 0x4002\_0000 (128 KB). The app
will be loaded by the firmware from the start. The app's stack grows
from the end (see [apps/libcrt0/crt0.S](apps/libcrt0/crt0.S)). A
larger app has tradeoff of having a smaller stack.
```